README.md

```diff
@@ -6,24 +6,19 @@ This is a mobile app project template using [Uno Platform](https://github.com/un
 
 We use `dotnet` project templates to easily create new projects. It simplifies the **project renaming** and supports **conditional inclusions**.
 
-### Installing and uninstalling the template
+### Installing the template
 
-1. In order to install the template, clone this repository on your machine and open a command prompt at the root of the project and run the following command 
+1. In order to install the template, clone this repository on your machine and open a command prompt at the root of the cloned project and run the following command 
```

Comment:
"clone" is already mentioned.

It should be:
In order to install the template, clone this repository on your machine, open a command prompt at its root, and run the following command